### DISCUSSION TITLE
Recommended Bootstrap Approach for Asp.Net Core

### DISCUSSION THREAD

**Discussion by mxmissile** [nhibernate/nhibernate-core#2729] [Q&A] [answered]
Is this still the recommended approach for using NHibernate and Asp.Net Core?

https://github.com/nhibernate/nhibernate-core/tree/master/src/NHibernate.Example.Web

**Comment by knuxbbs** [comment, marked as answer]
Yes, it's still fresh. And with a modern approach.
